src/test/scala/io/iohk/ethereum/blockchain/sync/StateSyncSpec.scala

```diff
@@ -75,6 +77,7 @@ class StateSyncSpec
   it should "sync state to different tries when peer provide mixed responses" in new TestSetup() {
     forAll(ObjectGenerators.genMultipleNodeData(1000)) { nodeData =>
       val initiator = TestProbe()
+      initiator.ignoreMsg { case SyncStateSchedulerActor.StateSyncStats(_, _) => true }
```

Comment:
this need to be added also in last test `start state sync when receiving start signal while bloom filter is loading`